src/scala/ly/stealth/mesos/kafka/Executor.scala

```diff
@@ -92,6 +95,25 @@ object Executor extends org.apache.mesos.Executor {
       }
     }
 
+    def startCollectingMetrics(): Unit = {
+      new Thread {
+        setName("BrokerMetrics")
+        def send(driver: ExecutorDriver, metrics: Broker.Metrics): Unit = {
+          driver.sendFrameworkMessage(JSONObject(Map("metrics" -> metrics.toJson)).toString().getBytes)
+        }
+        override def run(): Unit = {
+          while (true) {
+            try {
+              BrokerServer.Metrics.collect.foreach(metrics => send(driver, metrics))
+              Thread.sleep(5000L)
```

Comment:
We need a way to change this through the CLI and API without restarting the brokers (preferably but at a bare min not hardcoded value). Not everyone uses 5 sec as metric value